docs/architecture/modernize-with-azure-containers/modernize-existing-apps-to-cloud-optimized/when-to-deploy-windows-containers-to-azure-container-service-kubernetes.md

```diff
@@ -1,19 +1,21 @@
 ---
-title: When to deploy Windows Containers to Azure Container Service (that is, Kubernetes)
-description: Modernize existing .NET applications with Azure Cloud and Windows containers | When to deploy Windows Containers to Azure Container Service (that is, Kubernetes)
-ms.date: 12/12/2021
+title: When to deploy Windows Containers to Azure Kubernetes Service
+description: Modernize existing .NET applications with Azure Cloud and Windows containers | When to deploy Windows Containers to Azure Kubernetes Service
+ms.date: 02/21/2022
 ---
-# When to deploy Windows Containers to Azure Container Service (that is, Kubernetes)
+# When to deploy Windows Containers to Azure Kubernetes Service (AKS)
 
-Azure Container Service optimizes the configuration of popular open-source tools and technologies specifically for Azure. You get an open solution that offers portability both for your containers and for your application configuration. You select the size, the number of hosts, and the orchestrator tools. Azure Container Service handles the infrastructure for you.
+Azure Kubernetes Service (AKS) simplifies deploying a managed Kubernetes cluster in Azure by offloading the operational overhead to Azure. You get an open solution that offers portability both for your containers and for your application configuration. You select the size, the number of hosts, and the orchestrator tools. Azure Kubernetes Service handles the infrastructure for you, it also supports Windows Server containers alongside Linux containers for more details visit this [link](https://docs.microsoft.com/azure/aks/windows-faq)
```

Comment:
```suggestion
Azure Kubernetes Service (AKS) simplifies deploying a managed Kubernetes cluster in Azure by offloading the operational overhead to Azure. You get an open solution that offers portability both for your containers and for your application configuration. You select the size, the number of hosts, and the orchestrator tools. Azure Kubernetes Service handles the infrastructure for you. It also supports Windows Server containers alongside Linux containers. For more details on Windows OS support and the differences between Windows and Linux, read the [FAQ]https://docs.microsoft.com/azure/aks/windows-faq).
```

Reply:
@nishanil, thank you for your suggestion.